niri-ipc/src/lib.rs

```diff
@@ -220,6 +220,12 @@ pub enum Action {
         ///  Whether to show the mouse pointer by default in the screenshot UI.
         #[cfg_attr(feature = "clap", arg(short = 'p', long, action = clap::ArgAction::Set, default_value_t = true))]
         show_pointer: bool,
+
+        /// Path to save the screenshot to.
+        ///
+        /// If `None`, the screenshot is saved according to the `screenshot-path` config setting.
+        #[cfg_attr(feature = "clap", arg(short = 'f', long, action = clap::ArgAction::Set))]
```

Comment:
Maybe we don't need the short flag for these, especially since it doesn't match the first letter